Resources/Storage/GFAL2_StorageBase.py

```diff
@@ -434,14 +434,17 @@ def __getSingleFile( self, src_url, dest_file ):
     params.overwrite = True  # old gfal removed old file first, gfal2 can just overwrite it with this flag set to True
     params.src_spacetoken = self.spaceToken
 
-    params.checksum_check = True if self.checksumType else False
+    if disableChecksum:
```

Comment:
params.checksum_check = bool(self.checksumType and not disableChecksum)
(bool casting is maybe not needed)